From eb51d9e30941f64a83999be8d5a0bbfa20583bcc Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Fri, 17 Jul 2015 11:12:50 -0400
Subject: [PATCH 2/2] Remove spl warnings for unused features

1) Not using eMMC environment so don't look for one.
2) Not running falcon mode, so don't try to load kernel args.
---
 include/configs/am335x_evm.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index 98681a6..d1ae228 100644
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -417,6 +417,9 @@
 					"128k(u-boot-env2),3464k(kernel)," \
 					"-(rootfs)"
 #elif defined(CONFIG_EMMC_BOOT)
+#define CONFIG_ENV_IS_NOWHERE
+#undef CONFIG_SPL_OS_BOOT
+/*
 #undef CONFIG_ENV_IS_NOWHERE
 #define CONFIG_ENV_IS_IN_MMC
 #define CONFIG_SPL_ENV_SUPPORT
@@ -425,6 +428,7 @@
 #define CONFIG_ENV_OFFSET		0x0
 #define CONFIG_ENV_OFFSET_REDUND	(CONFIG_ENV_OFFSET + CONFIG_ENV_SIZE)
 #define CONFIG_SYS_REDUNDAND_ENVIRONMENT
+*/
 #endif
 
 /* SPI flash. */
-- 
2.1.4

